www/src/pages/index.astro

```diff
@@ -17,6 +17,9 @@ let lang = 'en';
   <head>
     <BaseHead title={title} description={description} permalink={permalink} />
     <link rel="stylesheet" href="/global.css" />
+    <link rel="preconnect" href="https://cdn.jsdelivr.net/">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lite-youtube-embed@0.2.0/src/lite-yt-embed.css" integrity="sha256-99PgDZnzzjO63EyMRZfwIIA+i+OS2wDx6k+9Eo7JDKo=" crossorigin="anonymous">
```

Comment:
We're discussing this in discord, agree would be good to vendor.

Reply:
I was trying to figure out how to do that... but I ran into problems importing the css from an npm module. 

```astro
---
import "lite-youtube-embed/src/lite-yt-embed.css";
---
```
![image](https://user-images.githubusercontent.com/10626596/130518986-2ba1ae9d-407d-4823-8c75-5a8ef2c10fce.png)

Reply:
hmm, maybe just as a `<link>` element on the page instead? Similar to what you have now

Reply:
`<link rel="stylesheet" href="lite-youtube-embed/src/lite-yt-embed.css"/>` doesn't seem to work either.

Reply:
oh right, because it's a package! What about literally copying that file into `public`, and then using it that way? Not too different from using jsdeliver, then